CMakeLists.txt

```diff
@@ -412,6 +412,8 @@ IF (WITH_MRUBY)
     ENDIF ()
     ADD_CUSTOM_TARGET(mruby MRUBY_TOOLCHAIN=${MRUBY_TOOLCHAIN} MRUBY_CONFIG=${CMAKE_CURRENT_SOURCE_DIR}/misc/mruby_config.rb MRUBY_BUILD_DIR=${CMAKE_CURRENT_BINARY_DIR}/mruby ruby minirake
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/deps/mruby)
+    ADD_CUSTOM_TARGET(mruby-test MRUBY_TOOLCHAIN=${MRUBY_TOOLCHAIN} MRUBY_CONFIG=${CMAKE_CURRENT_SOURCE_DIR}/misc/mruby_test_config.rb MRUBY_BUILD_DIR=${CMAKE_CURRENT_BINARY_DIR}/mruby-test ruby minirake
+        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/deps/mruby)
```

Comment:
Assuming that the only difference between `mruby_config.rb` and `mruby_test_config.rb` is the addition of `iij/mruby-mtest`, how about adding the library to `mruby_config.rb` instead of introducing a new build of mruby?

Or another approach would be to keep `mruby_config.rb` as-is and call `require` when running the unit tests (note: we already have https://github.com/h2o/h2o/tree/master/deps/mruby-require as a dependency).